src/containers/dashboard/index.js

```diff
@@ -68,15 +70,19 @@ class Dashboard extends React.Component {
               component={DashboardEditInfos}
             />
           )}
-          {this.state.render && (
+          {this.props.location === `${baseUrl}dashboard/payment/success` ? <DashboardPaymentSuccess/> : null}
```

Comment:
You should use `<Route path={baseUrl + 'dashboard/payment/success'} component={DashboardPaymentSuccess} />` instead of direct manipulation of `location`.
Same goes for all components here

Reply:
didnt knew about "exact" parameters for <Route> component